tf/edgeml/trainer/bonsaiTrainer.py

```diff
@@ -26,6 +25,7 @@ def __init__(self, bonsaiObj, lW, lT, lV, lZ, sW, sT, sV, sZ,
         '''
 
         self.bonsaiObj = bonsaiObj
+        self.RegLoss = regLoss
```

Comment:
self.regLoss, r is small everywhere when we use camel case of variables. @siddharthdivi